src/sentry/api/endpoints/sentry_app_requests.py

```diff
@@ -21,6 +21,19 @@ def format_request(self, request, sentry_app):
             "responseCode": request.get("response_code"),
         }
 
+        if "error_id" in request and "project_id" in request:
+            try:
+                project = Project.objects.get_from_cache(id=request["project_id"])
```

Comment:
@nolachen Do you know if we cache the miss as well? If someone puts in a bad project ID, I would not want it to keep querying it over and over and fail.

Reply:
@scefali no but this only gets called when you request the endpoint so I think it should be okay